.github/workflows/run-all-e2e-tests.yaml

```diff
@@ -0,0 +1,57 @@
+# Runs Azure tests when pull request opened, repopened or synchronized
+name: E2E Tests - Run all tests on PR
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [labeled, synchronize]
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  set-global-conditions:
+    runs-on: ubuntu-latest
+    outputs:
+      runs_e2e_tests: ${{ steps.set_runs_e2e_tests.outputs.RUNS_E2E_TESTS }}
+    if: |
+      github.event_name == 'pull_request' &&
+      (
+        (github.event.action == 'labeled' &&  github.event.label.name == 'runs-e2e-tests-temp') ||
+        (github.event.action == 'synchronize' && contains(github.event.pull_request.labels.*.name, 'runs-e2e-tests-temp'))
```

Comment:
Once PR is ready to merge, 'runs-e2e-tests-temp' will be renamed to 'runs-e2e-tests'